processor/src/main/java/com/beust/version/VersionProcessor.java

```diff
@@ -79,8 +79,10 @@ private String findVersionValue(Version version) throws IOException {
             if (f.exists()) {
                 log("Found " + f);
                 Properties p = new Properties();
-                p.load(new FileReader(f));
+                FileReader r = new FileReader(f);
```

Comment:
That works too, since you have `SourceVersion.RELEASE_8`. I'd personally use `RELEASE_6` so the annotator can be used with older projects.